### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/simdzone/actions/runs/5498219655#summary-14886605692

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/simdzone/actions/runs/5498219695#summary-14886606009

### CODE REVIEW

File: .github/workflows/coverity-scan.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# coverity-scan.yml -- Github Actions workflow for Coverity Scan analysis
+# coverity-scan.yml -- GitHub Actions workflow for Coverity Scan analysis
```
Comment (jsoref) [inline .github/workflows/coverity-scan.yml:2]: brand

File: FORMAT.md [hunk 2]
```diff
@@ -215,12 +215,12 @@ anything other than domain names and text strings, MUST not be quoted.
   "$DATE" and "$GENERATE" (and "$TTL" before RFC2308) are considered valid
   domain names in other implementations (based on what is accepted for domain
   names, see earlier points). It seems "$" is better considered a reserved
-  character (possibly limiting it's special special status to the start of the
+  character (possibly limiting its special status to the start of the
```
Comment (jsoref) [inline FORMAT.md:218]: two changes:
* removing a duplicated word which didn't seem to add value
* `it's` = `it is` which isn't the right concept here, replacing it with `its` (possessive)

**Review by jsoref — COMMENTED** [review]
Spelling fixes mostly suggested by Google Sheets. All fault mine.

**Comment by k0ekk0ek** [comment]
Hi @jsoref. Thanks! These are all good spelling fixes. Currently simdzone is not yet production quality and there's no documentation yet, but once we reach that point. It'd be very good to run this on a regular basis. Any way to have this included with the project eventually (possibly with some manual overrides?).

**Comment by jsoref** [comment]
Certainly. I'd be happy to make a PR for it. The commit would look something like this: https://github.com/jsoref/simdzone/commit/61ceb58588d735f2603ccaff13be0618b0767b73

I generally make a couple of changes when I make a PR for projects to use check-spelling… you can see that the workflow file in this commit is using `prerelease`, and I'd change that to the current release.

I also tend to restrict the update feature to forks.

And yes, the tool expects that any project will need overrides, they're generally contained in `expect.txt`. The README file for the configuration directory is designed to help people understand how to tune it.

I've started working on a user friendly view of the documentation for check-spelling: https://docs.check-spelling.dev/

And, of course, if you have questions, you can @ me and I'll try to answer them.

**Comment by k0ekk0ek** [comment]
Thanks! I'll review the PR when it comes in :+1:

**Pull request merged**